src/index.html

```diff
@@ -6,9 +6,47 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com"
+      crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo" >
+        <img class="image" src="images\logo.png"
+          alt="logo of moyo-header">
+      </a>
+      <nav class="nav">
+        <ul class="nav__ul">
+          <li>
+            <a class="nav__link is-active" href="#apple">APPLE</a>
```

Comment:
One thing I missed - content of links should be lowercase and you should style it to be uppercase using CSS.
So please add text-transform to `.nav__link {}`